pythonbits/bb.py

```diff
@@ -239,13 +274,25 @@ def _render_tv_specifier(self):
             return TvSpecifier(title, guess['season'],
                                guess.get('episode', None))
 
+        cguess = self['confirmed_guess']
+        log.debug('Confirmed guess: %s' % (cguess,))
+        if cguess['type'] == 'episode':
+            if self['title_arg']:
+                title = self['title_arg']
+            else:
+                title = cguess['title']
+            return TvSpecifier(title, cguess['season'],
+                               cguess.get('episode', None))
+
```

Comment:
not sure about the point of this

Reply:
I don't really remember, but I think there was a problem with auto-detecting stuff. (Wish I had the time to write tests for everything.)

Reply:
> 
> Wish I had the time to write tests for everything.

Ditto... Maybe I can get the tests working again.